front/src/applications/stdcmV2/components/StdcmResults/StdcmResults.tsx

```diff
@@ -73,6 +80,104 @@ const StcdmResults = ({
     );
   }, [selectedSimulation]);
 
+  const infraId = useInfraID();
+  const pathProperties = usePathProperties(
+    infraId,
+    conflictData.pathfinding_result as PathfindingResultSuccess,
+    ['operational_points', 'zones', 'geometry']
+  );
+
+  useEffect(() => {
```

Comment:
I think, the conflits should be treated in an other file (maybe `useStdcm`), set new response when `response.status === 'conflicts'` and update logic in `useStdcmResults` for stdcmResponse === `conflicts`